code/modules/mob/living/simple_animal/hostile/bees.dm

```diff
@@ -114,11 +114,11 @@
 
 	var/static/mutable_appearance/greyscale_overlay
 	greyscale_overlay = greyscale_overlay || mutable_appearance('icons/mob/bees.dmi')
-	greyscale_overlay.icon_state = "[icon_base]_grey"
+	greyscale_overlay.icon_state = "bee_grey"
 	greyscale_overlay.color = col
 	add_overlay(greyscale_overlay)
 
-	add_overlay("[icon_base]_wings")
```

Comment:
hmmmm

Reply:
I want to just use the wings from the standard for the queen sprite since they are the same and it meant using two less sprites. This should work right?